sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulse.java

```diff
@@ -20,7 +21,9 @@
 
 public class QuickPulse {
 
-    static final int QP_INVARIANT_VERSION = 1;
+    //supports Open Tel metrics but not live filtering
+    // change to 7 once live filtering is also supported
+    static final int QP_INVARIANT_VERSION = 6;
```

Comment:
Are the different invariant versions defined somewhere? If so, please link in this comment!

Reply:
These are not explicitly defined anywhere. I spoke to Ram and we decided to let 6 represent that Otel metrics are supported but not Live filtering, and let 7 represent that Otel metrics and Live filtering are supported. 

I am currently working on the UX such that it restricts customers to only select the "Metric" category when the invariant version is 6. Will link the PR here when I have it out.

Reply:
What would be the correct invariant version for live metrics filtering being supported, but not the otel metrics? The other SDKs will probably implement filtering first, before they implement the otel metrics functionality.

Reply:
That would be 5, which is what the .NET SDK is currently using to show that live filtering is enabled but not Otel metrics functionality.